src/redux/configureStore.js

```diff
@@ -1,9 +1,14 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
 import logger from 'redux-logger';
 import thunk from 'redux-thunk';
 import reducerRockets from './rockets/rockets';
 import reducerMissions from './missions/missions';
 
+const rootReducer = combineReducers({
```

Comment:
[OPTIONAL] Well done overall 💯 but consider fixing this in order to improve the testing process👇🏽 

![Screenshot 2023-04-30 at 00 01 34](https://user-images.githubusercontent.com/45739963/235334913-96f88299-b476-4a3a-83c2-573c0ce39dfb.png)

No changes are required because of it as the tests are still passing ✔️